crates/sui-graphql-rpc/src/server/version.rs

```diff
@@ -135,17 +142,39 @@ fn parse_version(version: &str) -> Option<(&str, &str)> {
 
 #[cfg(test)]
 mod tests {
+    use std::net::SocketAddr;
+
+    use super::*;
+    use crate::config::{ConnectionConfig, Version};
+    use crate::metrics::Metrics;
     use axum::{body::Body, middleware, routing::get, Router};
     use expect_test::expect;
+    use mysten_metrics;
     use tower::ServiceExt;
 
-    use super::*;
-
+    const RPC_VERSION_FULL: &str = env!("CARGO_PKG_VERSION");
```

Comment:
Better to just make a version up here, as it's a test -- it shouldn't matter right?

Reply:
As per the tests, this will fail if not passed the PKG version. I'll probably have to rework this stuff after `Version` type is changed.